src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -230,19 +232,21 @@ def test_write_stash(self):
         full_stash = {
             'blocked': ['fooo@baaaa:999'],
             'unblocked': [f'{self.five_ver.guid}:123.5']}
-        with open(newer_mlbf.stash_path) as stash_file:
+        with open(newer_mlbf._stash_path) as stash_file:
             assert json.load(stash_file) == full_stash
         assert newer_mlbf.stash_json == full_stash
 
     def test_should_reset_base_filter(self):
```

Comment:
Should this test case be renamed to reflect that it's also testing `blocks_changed_since_previous`?

Reply:
https://github.com/mozilla/addons-server/pull/14158/commits/ecdf64d6ae42063feacab214f5f4aff24141818c